pkg/sqs/service.go

```diff
@@ -19,31 +21,45 @@ type ServiceSettings struct {
 	AutoCreate bool
 }
 
+//go:generate mockery -name Service
+type Service interface {
+	CreateQueue(settings Settings) (*Properties, error)
+	QueueExists(name string) (bool, error)
+	GetTaggedQueues(tagsFilter map[string][]string) ([]Properties, error)
+	GetProperties(name string) (*Properties, error)
+	GetUrl(name string) (string, error)
+	GetArn(name string) (string, error)
+	Purge(name string) error
+}
+
 type service struct {
```

Comment:
That's good a thought. But iIn this case, I wanted to prevent exporting the actual struct, so only the interface is used outside of the package.
Furthmore, the package itself is already named `sqs`, thus I did not include it in the service name :)